assignments/arrays.js

```diff
@@ -112,26 +112,41 @@ console.log(carYears());
 
 // ==== Challenge 5 ====
 // The car lot manager needs to find out how many cars are older than the year 2000. Using the carYears array you just created, find out how many cars were made before the year 2000 by populating the array oldCars and logging it's length.
-let oldCars = [carYears()];
-
-// let carFunc = function(){
-//     let sort = [];
-//     for (let i = 0; i < oldCars.length; i++){
-//        if (oldCars[i]< 2000) {
-//            sort.push(oldCars[i]);
-//        }
-//         }
-//         return (sort);
-// };
+let oldCars = carYears();
+
+ let carFunc = function(){
```

Comment:
`let`, `const`, `var` allows us to create variable binding names, `let` and `var` are both mutable, meaning we can change them. most of the time though we don't want to have that effect especially when declaring functions. we would use `const` for most of the time when we declare these bindings. and if the variable does need to change over time then we would consider using `let`, but we should always default to `const`